documentation/guides/record-tests/README.md

```diff
@@ -101,17 +101,17 @@ You can do the following during test recording:
 
     When you add an on-page action or assertion, you can visually select a target element on the tested page. To do this, click the ![Element picker](../../../images/guides/element-picker-icon.png) button in the action parameters' area and click an element on the page.
 
-    You can also choose a desired property from the auto-generated property list and see the assertion status for assertion.
+    You can also choose a property from the auto-generated property list and see an assertion's status.
```

Comment:
@dirk-pieterse 
We rewrite this phrase in another PR.

Reply:
Yes, but  You can also choose a **desired** property from the auto-generated property list and see **the assertion status for assertion**. 
'desired' is unnecessary 
'the assertion status for assertion' does not make sense

Reply:
I'll rewrite this sentence as it is in another PR, this will avoid conflicts in the file